.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    types: [ opened, edited, reopened, synchronize ]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-18.04
```

Comment:
seems ubuntu-18.04 will be deprecated soon https://github.com/actions/runner-images/issues/6002 
use ubuntu-latest instead?

Reply:
"ubuntu-latest" refers to version 22.04, in which cgroup v2 is enabled by default and causes some unit tests to fail. Therefore, I switched the system version to the older 18.04. However, I also tried version 20.04 and it worked as well, which I consider to be a viable option.